bycycle/plts/burst.py

```diff
@@ -0,0 +1,227 @@
+"""Plot burst detection parameters."""
+
+from itertools import cycle
+
+import numpy as np
+from scipy.stats import zscore
+
+import matplotlib.pyplot as plt
+
+from neurodsp.plts import plot_time_series, plot_bursts
+
+from bycycle.plts.cyclepoints import plot_cyclepoints
+from bycycle.plts.utils import apply_tlims, get_extrema
+
+###################################################################################################
+###################################################################################################
+
+def plot_burst_detect_summary(df, sig, fs, osc_kwargs, tlims=None, figsize=(15, 3),
+                              plot_only_result=False, burst_params=None, interp=True):
+    """
+    Create a plot to study how the cycle-by-cycle burst detection
+    algorithm determines bursting periods of a signal.
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        Dataframe output of :func:`~.compute_features`.
+    sig : 1d array
+        Time series to plot.
+    fs : float
+        Sampling rate, in Hz.
+    osc_kwargs : dict
+        Burst parameter keys and threshold value pairs, as defined in the 'burst_detection_kwargs'
+        argument of :func:`.compute_features`.
+    tlims : tuple of (float, float), optional, default: None
+        Start and stop times for plot.
+    figsize : tuple of (float, float), optional, default: (15, 3)
+        Size of each plot.
+    plot_only_result : bool, optional, default: False
+        Plot only the signal and bursts, excluding burst parameter plots.
+    burst_params : list of str, optional, default: None
+        The names of the ``df`` burst detection parameter columns. These names may differ from the
+        keys in ``osc_kwargs``. This is only needed for non-default burst parameters.
+    interp : bool
```

Comment:
optional, default: True

Also, I think it could be a little more descriptive, like:
"If True, interpolates between given values. Otherwise, plots in a step-wise fashion."